pyproject.toml

```diff
@@ -100,10 +94,11 @@ sphinx-notfound-page = {version="*", optional = true}
 sphinx-tabs = {version="*", optional = true}
 nbdime = {version="*", optional = true}
 myst-parser = {version="*", optional = true}
-optax = {version="*", optional = true}
+optax = {version=">=0.2.2", optional = true}
 signac = {version="*", optional = true}
+flax = {version=">=0.8.2", optional = true}
 sax = {version="*", optional = true}
-vtk = {version="<=9.2.6", optional = true}
+vtk = {version=">=9.2.6", optional = true}
```

Comment:
Similarly, are you sure about this? @dbochkov any thoughts?

Reply:
Should be ok starting with 2.7